nb2kg/nb2kg/managers.py

```diff
@@ -113,7 +113,7 @@ def start_kernel(self, kernel_id=None, path=None, **kwargs):
                 method='POST',
                 body=json_encode({
                     'name': kernel_name,
-                    'env': {k:v for (k,v) in dict(os.environ).items() if k.startswith('KERNEL_')}
+                    'env': {k:v for (k,v) in dict(os.environ).items() if k.startswith('KERNEL_') or k in os.environ['KG_WHITELIST'].split(":")}
```

Comment:
I would propose changes to maintain consistency with KG:

* [KG uses `KG_ENV_WHITELIST`](https://github.com/jupyter/kernel_gateway/blob/595e34ababe24f88697968c7deb7de735760a6ae/kernel_gateway/jupyter_websocket/__init__.py#L33), so use that name instead of using a new name `KG_WHITELIST` for the same variable.
* [KG uses comma `,` as delimiter](https://github.com/jupyter/kernel_gateway/blob/595e34ababe24f88697968c7deb7de735760a6ae/kernel_gateway/jupyter_websocket/__init__.py#L40), not `:`.

Other than that, looks fine.